packages/atst/README.md

```diff
@@ -28,42 +28,50 @@ Install
 npm install @eth-optimism/atst wagmi @wagmi/core ethers@5.7.0
 ```
 
-## atst typescript sdk
+## atst typescript/javascript sdk
```

Comment:
nit redundant could jut call it a typescript sdk or a javascript sdk

Reply:
@zainbacchus asked for it to be both.